requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.10, <1.24
 matplotlib >=3.1.1, !=3.3.0rc1
 PyYAML >=5.1, !=*rc*
+pymatreader
```

Comment:
```suggestion
```

`pymatreader` is not a mandatory requirement: `phys2bids` installation is modular, so if you want to be able to read matlab file, you need to install `pymatreader`, if not you don't. same thing for Acqknowledge or spike2 files. Similarly, your tests might be failing because you'd need to install physbids with either the `dev` or the `all` option
(see [here](https://phys2bids.readthedocs.io/en/latest/contributing.html#full-developer-installation)).

It's true that we did not update the documentation accordingly, so thank you for pointing it out! I will open an issue about it.